libs/community/langchain_community/document_loaders/parsers/pdf.py

```diff
@@ -277,8 +278,10 @@ def lazy_parse(self, blob: Blob) -> Iterator[Document]:  # type: ignore[valid-ty
 
             yield from [
                 Document(
-                    page_content=page.get_text(**self.text_kwargs)
-                    + self._extract_images_from_page(doc, page),
+                    page_content=(lambda content: (
```

Comment:
Great! But it would be nice to improve some code readability.

Reply:
Agree! Put it in two seperate methods.